diff --git a/node_modules/aframe-extras/src/pathfinding/nav-agent.js b/node_modules/aframe-extras/src/pathfinding/nav-agent.js
index 21c8907..e15ddb6 100644
--- a/node_modules/aframe-extras/src/pathfinding/nav-agent.js
+++ b/node_modules/aframe-extras/src/pathfinding/nav-agent.js
@@ -2,7 +2,8 @@ module.exports = AFRAME.registerComponent('nav-agent', {
   schema: {
     destination: {type: 'vec3'},
     active: {default: false},
-    speed: {default: 2}
+    speed: {default: 2},
+    lookAtTarget: {default: true} // added line from README
   },
   init: function () {
     this.system = this.el.sceneEl.systems.nav;
@@ -80,7 +81,8 @@ module.exports = AFRAME.registerComponent('nav-agent', {
 
       // Look at the next waypoint.
       gazeTarget.y = vCurrent.y;
-      el.object3D.lookAt(gazeTarget);
+      if (data.lookAtTarget) // added from README
+        el.object3D.lookAt(gazeTarget);
 
       // Raycast against the nav mesh, to keep the agent moving along the
       // ground, not traveling in a straight line from higher to lower waypoints.
